modules/luxon.js

```diff
@@ -0,0 +1,6987 @@
+/* eslint-disable prefer-rest-params */
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable no-useless-escape */
+/* eslint-disable no-loop-func */
+/* eslint-disable guard-for-in */
+/* eslint-disable brace-style */
+/* eslint-disable no-use-before-define */
+/* eslint-disable prefer-const */
+/* eslint-disable no-restricted-globals */
+/* eslint-disable no-unused-vars */
+/* eslint-disable class-methods-use-this */
+/* eslint-disable no-plusplus */
+/* eslint-disable no-continue */
+/* eslint-disable default-case */
+/* eslint-disable no-nested-ternary */
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable max-classes-per-file */
+/* eslint-disable max-len */
+// these aren't really private, but nor are they really useful to document
```

Comment:
- [ ] Please don't deactivate all the linters, instead search on how to make the luxon library works on minimum codes [take a look here](https://moment.github.io/luxon/#/?id=luxon)